lib/ansible/modules/source_control/git.py

```diff
@@ -42,10 +42,10 @@
     dest:
         required: true
         description:
-            - Absolute path of where the repository should be checked out to.
```

Comment:
The reason is we don't really want to specifically call out relative paths as what they are relative to is determined by the setup of the remote system rather than anything under ansible's control.